Messages.py

```diff
@@ -1374,31 +1180,32 @@ def update_warp_song_text(messages: list[Message], world: World) -> None:
         0x4004: 'LH Owl Flight -> Hyrule Field',
     }
 
-    for id, entr in msg_list.items():
-        if 'warp_songs_and_owls' in world.settings.misc_hints or not world.settings.warp_songs:
-            destination = world.get_entrance(entr).connected_region
-            destination_name = HintArea.at(destination)
-            color = COLOR_MAP[destination_name.color]
-            if destination_name.preposition(True) is not None:
-                destination_name = f'to {destination_name}'
-        else:
-            destination_name = 'to a mysterious place'
-            color = COLOR_MAP['White']
+    if world.settings.logic_rules != "glitched": # Entrances not set on glitched logic so following code will error
```

Comment:
This is outdated now with the new advanced logic. You'll probably need to pull the latest dev changes and merge/update instances like this.